com/duke/parser/Parser.java

```diff
@@ -0,0 +1,151 @@
+package com.duke.parser;
+
+import java.time.LocalDate;
+
+public class Parser {
```

Comment:
Should there be a javadoc description for the class? I noticed this in the other classes also.